express/scripts/scripts.js

```diff
@@ -128,6 +129,10 @@ const miloLibs = setLibs(LIBS);
   document.head.append(footerMeta);
 
   listenMiloEvents();
+  if (getMetadata('sheet-powered') === 'Y') {
```

Comment:
This approach is definitely much better. Nice one 🙌 
It should be moved to the end of the decorateArea method though in our `utils.js`.

Reply:
We are competing with Milos own placeholder replacement script, so this has to be loaded earlier than that. Moving to decorateArea seems too early

Reply:
I'm not understanding you. What do you mean? 
You're saying that this has to be loaded super early but that decorateArea seems too early. 
The reason I'm saying that you should move this to decorateArea is so that content-replace also runs on fragments.  The big problem that we really have is that decorateArea is an asynchronous method
If you don't know what I mean we can jump on a call and I can run you through it.

Reply:
Sorry, I meant to say loaded too late. When I tested it locally Milo had already replaced the placeholders with its own. As you said, migrating to using [[ ]] is a solution, but we already ran into issues with the placeholders having different column names in Milo vs ours. Working with a single sharepoint is going to continue resulting in issues like this. 

decorateAreas will have to become async if we move the content replace there unless we find a replacement for getMetadata. I would definitely appreciate going over the impact changing decorateAreas to async would have tomorrow.

Reply:
BTW @vhargrave , just updated the PR with the moved content replace script